integration/querier_remote_read_test.go

```diff
@@ -127,10 +133,48 @@ func TestQuerierRemoteRead(t *testing.T) {
 	require.Len(t, resp.Results, 1)
 	require.Len(t, resp.Results[0].Timeseries, 1)
 	require.Len(t, resp.Results[0].Timeseries[0].Labels, 1)
-	require.Equal(t, "series_1", resp.Results[0].Timeseries[0].Labels[0].GetValue())
-	require.Len(t, resp.Results[0].Timeseries[0].Samples, 1)
-	require.Equal(t, int64(expectedVectors[0].Timestamp), resp.Results[0].Timeseries[0].Samples[0].Timestamp)
-	require.Equal(t, float64(expectedVectors[0].Value), resp.Results[0].Timeseries[0].Samples[0].Value)
+	require.Equal(t, seriesName, resp.Results[0].Timeseries[0].Labels[0].GetValue())
+	isSeriesFloat := len(resp.Results[0].Timeseries[0].Samples) == 1
+	isSeriesHistogram := len(resp.Results[0].Timeseries[0].Histograms) == 1
+	require.Equal(t, isSeriesFloat, !isSeriesHistogram)
+	if isSeriesFloat {
+		require.Equal(t, int64(expectedVectors[0].Timestamp), resp.Results[0].Timeseries[0].Samples[0].Timestamp)
+		require.Equal(t, float64(expectedVectors[0].Value), resp.Results[0].Timeseries[0].Samples[0].Value)
+	} else if isSeriesHistogram {
+		isEqualSampleAndHistogram(t, expectedVectors[0], resp.Results[0].Timeseries[0].Histograms[0])
+	}
+}
+
+func isEqualSampleAndHistogram(t *testing.T, expectedVector *model.Sample, histogram prompb.Histogram) {
```

Comment:
don't we have a utility function for this already? we should

Reply:
Couldn't find one, did you? I think our utility functions are for mimirpb. prompb doesn't have one because they don't import model from `prometheus/common`, and I don't see us comparing these 2 types anywhere else.

Reply:
Replaced this with a conversion to same type